pages/linux/ldapsearch.md

```diff
@@ -0,0 +1,35 @@
+# ldapsearch
+
+> CLI utility for querying an LDAP directory.
+
+- Query LDAP for all items that match filter "{{memberOf}}={{group1}}" and return {{displayName}}:
+
+`ldapsearch -D '{{admin_DN}}' -w '{{password}}' -h {{ldap_host}} -b {{base_ou}} '{{memberOf}}={{group1}}' {{displayName}}`
+
+- Query LDAP with a no-newline password file and return {{displayName}}:
```

Comment:
Perhaps this will read better?
```suggestion
- Query an LDAP server with a no-newline password file and return the display name:
```

Reply:
Placeholder.

Changes here will take a cue from line 5's changes.

Reply:
Line now reads "Query an LDAP server with a no-newline password file for all items that are a member of the given group and return the object's {{displayName}} value:"

I'd be happy for a suggestion on phrasing, but tripped over the newline issue myself during testing.